cpp/include/cudf/strings/findall.hpp

```diff
@@ -72,11 +72,10 @@ std::unique_ptr<table> findall(
  *  [ ["b"]
  *    ["a","b","b"]
  *    ["a"]
- *    null ]
+ *    [] ]
  * @endcode
  *
- * A null output row results if the pattern is not found in the corresponding row
- * input string.
+ * A null output row occurs if the corresponding input row is null.
```

Comment:
Maybe add a small blurb about things that don't match resulting in an empty list. It is shown in the example, but might be worth calling out.